src/module/errorCheckConfig.ts

```diff
@@ -16,7 +16,10 @@ export default function (config: IEasyFirestoreModule): boolean {
       errors.push(`Missing \`${prop}\` in your module!`)
     }
   })
-  if (/(\.|\/)/.test(config.statePropName)) {
+  if (config.statePropName !== null && !isString(config.statePropName)) {
+    errors.push('statePropName must be null or a string')
```

Comment:
Can you change to:
`'statePropName must be null or a string without special characters'`